monkey/infection_monkey/utils/brute_force.py

```diff
@@ -0,0 +1,25 @@
+from itertools import chain, product
+from typing import Any, Iterable, Tuple
+
+
+def generate_identity_secret_pairs(
+    identities: Iterable, secrets: Iterable
+) -> Iterable[Tuple[Any, Any]]:
+    return product(identities, secrets)
+
+
+def generate_username_password_or_ntlm_hash_combinations(
```

Comment:
maybe `generate_credential_combinations` would be better? Methods with 4 parameters are already a mess, let's not re-type those parameters in the names as well.